### ISSUE TITLE
May I ask is there any code for the training?

### ISSUE THREAD

**Issue by D-Mad** [deepmind/tapnet#1]
impressive!
May I ask is there any code for the training?

**Comment by cdoersch** [comment]
Yes, there's training code based on Jaxline.  See here: https://github.com/deepmind/tapnet#installation-for-tap-net-training-and-inference
